doc/source/contributing.rst

```diff
@@ -527,7 +527,7 @@ unused function. However, style-checking the diff will not catch this because
 the actual import is not part of the diff. Thus, for completeness, you should
 run this command, though it will take longer::
 
-   git diff master --name-only -- '*.py' | grep 'pandas' | xargs -r flake8
+   git diff master --name-only -- '*.py' | grep 'pandas/' | xargs -r flake8
```

Comment:
not sure if its worth noting but need to remove the ``-r`` on osx :<

otherwise this command looks good to me (I use it!)